Assets/Scripts/PlayerController.cs

```diff
@@ -32,29 +32,35 @@ private void Update()
     {
         if (Input.anyKey)
         {
```

Comment:
Could we remove this Input.anyKey, seems to add unnecessary nested blocks.

Reply:
it makes sure that things only happen if a button is pressed. Otherwise, GetAxis("Horizontal") and "Vertical" slowly revert to neutral, changing the player facing back to normal as well.

Reply:
Hmm good point. Maybe we can use `Input.GetAxisRaw` instead, (i've tried locally, seems to be fine, let me know if i missed something). So we don't have the smoothing. Also could you change `https://github.com/this-place/this.place/pull/12/commits/e07d15eaa4963d48e353ab4286a577f3da2ca963` to `style:`